README.md

```diff
@@ -1,3 +1,329 @@
 # Bio-Formats JACE C++ bindings
 
 [![Build Status](https://travis-ci.org/ome/bio-formats-jace.png)](https://travis-ci.org/ome/bio-formats-jace)
+
+Note that this is a legacy project and is no longer actively maintained. Use
+at your own risk.
+
+##Introduction
+
+To make Bio-Formats accessible to software written in C++, we have
+created a Bio-Formats C++ interface. It uses
+[LOCI's jar2lib program](http://loci.wisc.edu/software/jar2lib) to generate a
+C++ proxy class for each equivalent Bio-Formats Java class. The resulting
+proxies are then compiled into a library, which represents the actual
+interface from C++ to Bio-Formats. Using this library in your projects gives
+you access to the image support of Bio-Formats.
+
+The JACE C++ bindings come with some standalone examples which you can use as
+a starting point in your own project:
+
+- [showinf](https://github.com/ome/bio-formats-jace/blob/master/src/main/cppwrap/showinf.cpp)
```

Comment:
We should be able to use relative links instead of the URL here i.e.  [showinf](src/main/cppwrap/showinf.cpp)